auth-web/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth-web",
-  "version": "1.0.12",
+  "version": "1.0.13",
```

Comment:
version "version": "1.0.13", is main branch now. please merge main to this branch and update to 14

Reply:
Okay. Let me fix it.